examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -73,19 +51,7 @@
           128,
           128
         ],
-        "availabilityProfile": "AvailabilitySet",
-        "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME"
-      },
-      {
-        "name": "agentsa",
```

Comment:
would be great to continue testing multiple agent pools, even if this one isn't storageAccount anymore

Reply:
We do that in the Windows tests (they are hybrid clusters)

Reply:
Good point. Let's make sure we still cover multiple Linux agents _somewhere_ (ie. jenkins)